internal/controller/isbservicerollout/isbservicerollout_controller.go

```diff
@@ -156,7 +157,7 @@ func (r *ISBServiceRolloutReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 	// Update the resource definition (everything except the Status subresource)
 	if r.needsUpdate(isbServiceRolloutOrig, isbServiceRollout) {
-		if err := r.client.Patch(ctx, isbServiceRollout, client.MergeFrom(isbServiceRolloutOrig)); err != nil {
+		if err := r.client.Update(ctx, isbServiceRollout); err != nil {
 			r.ErrorHandler(ctx, isbServiceRollout, err, "UpdateFailed", "Failed to patch isb service rollout")
```

Comment:
I guess maybe this should say "Failed to update" instead of "Failed to patch" now

Reply:
oh, Yeah, I'll update